runtime/ops/runtime.rs

```diff
@@ -32,8 +32,8 @@ fn op_main_module(state: &mut OpState) -> Result<String, AnyError> {
 /// This is an op instead of being done at initialization time because
 /// it's expensive to retrieve the ppid on Windows.
 #[op2(fast)]
-#[bigint]
```

Comment:
You can use `#[number]` here to force it to return an integer-like number up to `MAX_SAFE_INTEGER`